tagstudio/src/qt/widgets/video_player.py

```diff
@@ -0,0 +1,382 @@
+        # )
+        # self.album_art.setOpacity(0.0)
+        # Set up the buttons.
+        self.play_pause = QSvgWidget("./tagstudio/resources/pause.svg")
+        self.play_pause.setAttribute(Qt.WidgetAttribute.WA_TranslucentBackground, True)
+        self.play_pause.setMouseTracking(True)
+        self.play_pause.installEventFilter(self)
+        self.scene().addWidget(self.play_pause)
+        self.play_pause.resize(100, 100)
+        self.play_pause.move(
+            self.width() / 2 - self.play_pause.size().width() / 2,
+            self.height() / 2 - self.play_pause.size().height() / 2,
+        )
+        self.play_pause.hide()
+
+        self.mute_button = QSvgWidget("./tagstudio/resources/volume_muted.svg")
+        self.mute_button.setAttribute(Qt.WidgetAttribute.WA_TranslucentBackground, True)
+        self.mute_button.setMouseTracking(True)
+        self.mute_button.installEventFilter(self)
+        self.scene().addWidget(self.mute_button)
+        self.mute_button.resize(40, 40)
+        self.mute_button.move(
+            self.width() - self.mute_button.size().width() / 2,
+            self.height() - self.mute_button.size().height() / 2,
+        )
+        self.mute_button.hide()
+        # self.fullscreen_button = QSvgWidget('./tagstudio/resources/pause.svg', self)
+        # self.fullscreen_button.setMouseTracking(True)
+        # self.fullscreen_button.installEventFilter(self)
+        # self.scene().addWidget(self.fullscreen_button)
+        # self.fullscreen_button.resize(40, 40)
+        # self.fullscreen_button.move(self.fullscreen_button.size().width()/2, self.height() - self.fullscreen_button.size().height()/2)
+        # self.fullscreen_button.hide()
+
+        self.setContextMenuPolicy(Qt.ContextMenuPolicy.ActionsContextMenu)
+        self.opener = FileOpenerHelper(filepath=self.filepath)
+        autoplay_action = QAction("Autoplay", self)
+        autoplay_action.setCheckable(True)
+        autoplay_action.setChecked(
+            self.driver.settings.value("autoplay_videos", True, bool)
+        )
+        autoplay_action.triggered.connect(lambda: self.toggleAutoplay())
+        self.addAction(autoplay_action)
+        self.autoplay = autoplay_action
+
+        open_file_action = QAction("Open file", self)
+        open_file_action.triggered.connect(self.opener.open_file)
+        open_explorer_action = QAction("Open file in explorer", self)
+        open_explorer_action.triggered.connect(self.opener.open_explorer)
+        self.addAction(open_file_action)
+        self.addAction(open_explorer_action)
+
+    def close(self, *args, **kwargs) -> None:
+        self.player.stop()
+        super().close(*args, **kwargs)
+
+    def toggleAutoplay(self) -> None:
+        self.driver.settings.setValue("autoplay_videos", self.autoplay.isChecked())
+
+    def checkMediaStatus(self, media_status: QMediaPlayer.MediaStatus) -> None:
+        logging.info(media_status)
```

Comment:
Recommend commenting out this log
```suggestion
        # logging.info(media_status)
```